server/test/unit/server/webservices/test_urls.py

```diff
@@ -4,6 +4,9 @@
 
 
 class TestDjangoPluginsUrls(unittest.TestCase):
+    """
+    Test url matching for /plugins/* urls.
```

Comment:
I probably wouldn't write the URL here specifically in case it changes in the future.